net/wfb-ng/Makefile

```diff
@@ -0,0 +1,49 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wfb-ng
+PKG_VERSION:=24.09.23
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/svpcom/wfb-ng/tar.gz/$(PKG_NAME)-$(PKG_VERSION)?
+PKG_HASH:=c6cf669090952f6e430166a23b2fd787c6f7d4fd2d8d1996e8e3713f5cc4f8d9
+
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Vasily Evseenko <svpcom@p2ptech.org>
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+MAKE_FLAGS += VERSION=$(PKG_VERSION)-$(PKG_RELEASE) COMMIT=openwrt all_bin
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
I'm guessing this is what results in `/builder/build_dir/target-aarch64_generic_musl/wfb-ng-wfb-ng-24.09.23/` and then some tasks at the end of the build process failing? Have you tried `PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)`

Reply:
@stangri Yes. This is done due to github auto packaging issue. It automatically create archive for release with following structure:
```
$ tar tzvf wfb-ng-24.08.2.tar.gz  | head
drwxrwxr-x root/root         0 2024-10-01 16:49 wfb-ng-wfb-ng-24.08.2/
drwxrwxr-x root/root         0 2024-10-01 16:49 wfb-ng-wfb-ng-24.08.2/.github/
drwxrwxr-x root/root         0 2024-10-01 16:49 wfb-ng-wfb-ng-24.08.2/.github/ISSUE_TEMPLATE/
...
```

It concatenate project name + tag name (which is wfb-ng-24.08.2 in this case)

Reply:
I see. I thought there was a way to override this from the Makefile as I've had to deal with a similarly weirdly named archive content before, but maybe I'm misremembering things.

Reply:
Hello, any updates on this package? The OpenIPC fpv project is getting increasing interest and many requests when the package will be ready for easy deployment. I have 4 different router models up and running myself with the latest makefile, but it's a ... shitparty ... to compile manually from sources :)

Reply:
Can anyone respond to what is blocking merging this package? Thanks.

Reply:
please move `PKG_BUILD_DIR` below `PKG_HASH`.

Reply:
@1715173329 done, thanks!